index.html

```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Test</title>
+  </head>
+</html>
```

Comment:
Adauga si tag-ul de body
```suggestion
<html>
  <head>
    <title>Test</title>
  </head>
  
  <body>
  </body>
</html>
```